### PR TITLE
Update strudel to 0.9.0

### PR DESCRIPTION
**Pull request by felixroos**
+ load all default sounds

(this PR branches off https://github.com/munshkr/flok/pull/181 so that one should be merged first)

### CODE REVIEW

File: packages/web/src/lib/strudel-wrapper.ts [hunk 1]
```diff
@@ -63,18 +77,47 @@ export class StrudelWrapper {
...
+  return Promise.all([
+    samples(`${ds}/tidal-drum-machines.json`),
+    samples(`${ds}/piano.json`),
+    samples(`${ds}/Dirt-Samples.json`),
+    samples(`${ds}/EmuSP12.json`),
+    samples(`${ds}/vcsl.json`),
+  ]);
+}
+
+// this is a little bit awkward but the piano function has to be duplicated here..
```
Comment (munshkr) [inline packages/web/src/lib/strudel-wrapper.ts:109]: Why is this needed? 🤔

**Review by munshkr — APPROVED** [review]

File: packages/web/src/lib/strudel-wrapper.ts (hunk 1 above)
Comment (felixroos) [reply packages/web/src/lib/strudel-wrapper.ts:109]: it's not part of a strudel package because it expects the "piano" sound to be loaded, which does not happen by default. but maybe I'll just add it to strudel directly so it won't be needed here. The general idea of this wrapper here was that code from the strudel repl and flok can be copy pasted without breaking

Comment (munshkr) [reply packages/web/src/lib/strudel-wrapper.ts:109]: it's ok just wondering, makes sense to have sensible defaults here instead of the core package
